pandas/util/_test_decorators.py

```diff
@@ -187,3 +187,10 @@ def decorated_func(func):
                                    "installed->{installed}".format(
                                        enabled=_USE_NUMEXPR,
                                        installed=_NUMEXPR_INSTALLED))
+
+
+def parametrize_fixture_doc(*args):
```

Comment:
this is nice, in a future PR can you document this itself?

Reply:
can you add docs on this here

Reply:
@jreback done
sorry didn't see this comment at first, so I did two updates (one for rebase and one for doc), I think previous builds can be cancelled